### PR TITLE
Replaced qApp->translation to qtrc

### PR DESCRIPTION
**Pull request by igorkorsukov**
I would be clean up translations contexts, but not easy, so if it doing, then need separate pr

### CODE REVIEW

File: src/engraving/libmscore/cmd.cpp [hunk 1]
```diff
@@ -26,6 +26,10 @@
 */
 
 #include <assert.h>
+#include <QGuiApplication>
```
Comment (igorkorsukov) [inline src/engraving/libmscore/cmd.cpp:29]: needs for `qApp->keyboardModifiers()`

File: src/engraving/libmscore/mscore.cpp [hunk 2]
```diff
@@ -21,8 +21,11 @@
  */
 
 #include <QDir>
+#include <QCoreApplication>
```
Comment (igorkorsukov) [inline src/engraving/libmscore/mscore.cpp:24]: needs for `QCoreApplication::applicationDirPath()`

File: src/engraving/libmscore/barline.h [hunk 3]
```diff
@@ -144,7 +142,6 @@ class BarLine final : public Element
     static BarLineType barLineType(const QString&);
 
     int subtype() const override { return int(_barLineType); }
-    QString subtypeName() const override { return qApp->translate("barline", barLineTypeName().toUtf8()); }
```
Comment (vpereverzev) [inline src/engraving/libmscore/barline.h:147]: Has it not been used?

**Review by vpereverzev — APPROVED** [review]

File: src/engraving/libmscore/barline.h (hunk 3 above)
Comment (igorkorsukov) [reply src/engraving/libmscore/barline.h:147]: yep

**Pull request merged**